Gemfile.lock

```diff
@@ -63,12 +63,13 @@ GEM
       io-like (~> 0.3.0)
     arel (9.0.0)
     ast (2.4.0)
-    authlogic (4.4.2)
-      activerecord (>= 4.2, < 5.3)
-      activesupport (>= 4.2, < 5.3)
+    authlogic (5.0.0)
```

Comment:
Not sure if this was intentional, but this PR deals is open for the authlogic bump: https://github.com/sudara/alonetone/pull/413

Reply:
That authlogic PR exibits the `db:seeds` problem, so un-bumping here should get tests passing again and we can deal with validations and the authlogic upgrade separately.

Reply:
Sounds good.